### PR TITLE
MAUI-4553 Built In Views UG for Shimmer in .NET MAUI 

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- I have added the Built in views UG content in .NET MAUI

### CODE REVIEW

File: MAUI/Shimmer/built-in-views.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+platform: maui
+control: SfShimmer
+documentation: ug
+---
+
+# Built-in Views of .NET MAUI Shimmer (SfShimmer)
+
+## Built-in types
+
+Syncfusion .NET MAUI Shimmer (SfShimmer) control provides seven different built-in shimmer view types.
```
Comment (florenceliliana) [inline MAUI/Shimmer/built-in-views.md:14]: Syncfusion .NET MAUI Shimmer (SfShimmer) control provides seven built-in shimmer view types.

File: MAUI/Shimmer/built-in-views.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* `CirclePersona`
+* `SquarePersona`
+* `Profile`
+* `Article`
+* `Video`
+* `Feed`
+* `Shopping`
+
+The built-in shimmer types can be used by setting the `Type` of `SfShimmer`. By default, `CirclePersona` is initially rendered.
```
Comment (florenceliliana) [inline MAUI/Shimmer/built-in-views.md:24]: The built-in shimmer types can be used by setting the `Type` of `SfShimmer.` By default, the `CirclePersona` is initially rendered.

File: MAUI/Shimmer/built-in-views.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: About .NET MAUI Shimmer control | Syncfusion
+description: Learn here about the Built-in types support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.
```
Comment (florenceliliana) [inline MAUI/Shimmer/built-in-views.md:4]: description: Learn here about the Built-in types of support of Syncfusion .NET MAUI Shimmer (SfShimmer) control and more.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Shimmer/built-in-views.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/built-in-views.md]: Contents modified

File: MAUI/Shimmer/built-in-views.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Shimmer/built-in-views.md]: Contents modified

**Review by Sasikumar3595 — APPROVED** [review]
Code changes are fine

**Pull request closed without merging**